nextflow_schema.json

```diff
@@ -601,9 +601,28 @@
                 "mapping_bowtie2_maxins": {
                     "type": "integer",
                     "default": 500,
-                    "description": "Specify the maximum fragment length for Bowtie 2 paired-end mapping mode only.",
-                    "help_text": "Specify the maximum fragment length for valid paired-end alignments in Bowtie 2. This only applies to paired-end mapping (i.e. unmerged) and is therefore typically only useful for modern data.\n\n> Modifies Bowtie 2 parameter: `--maxins`",
-                    "fa_icon": "fas fa-filter"
+                    "description": "Specify the maximum fragment length for Bowtie2 paired-end mapping mode only.",
+                    "help_text": "The maximum fragment for valid paired-end alignments. Only for paired-end mapping (i.e. unmerged), and therefore typically only useful for modern data.\n\n> Modifies Bowtie2 parameter: `--maxins`",
+                    "fa_icon": "fas fa-exchange-alt"
+                },
+                "mapping_circularmapper_elongation_factor": {
+                    "type": "integer",
+                    "default": 500,
+                    "description": "Specify the number of bases to extend reference by (circularmapper only).",
+                    "help_text": "The number of bases to extend the reference genome with. By default this is set to 500 if not specified otherwise.",
+                    "fa_icon": "fas fa-external-link-alt"
+                },
+                "mapping_circularmapper_elongated_fasta": {
+                    "type": "string",
+                    "description": "Specify an elongated reference FASTA to be used for circularmapper.",
+                    "help_text": "Specify an already elongated FASTA file for circularmapper to avoid reprocessing.",
+                    "fa_icon": "fas fa-address-book"
+                },
+                "mapping_circularmapper_elongated_fai": {
+                    "type": "string",
+                    "description": "Specify a samtools index for the elongated FASTA file.",
+                    "help_text": "Specify the index for an already elongated FASTA file to avoid reprocessing.",
```

Comment:
```suggestion
                    "help_text": "Specify the index for an already elongated FASTA file to avoid reprocessing.",
```
```suggestion
                    "help_text": "Specify the index for an already elongated FASTA file to avoid regeneration.",
```